src/domain-services/location/location-service.ts

```diff
@@ -1,23 +1,33 @@
+import { Database } from '@unocha/hpc-api-core/src/db/type';
 import { Service } from 'typedi';
-import { DbModels } from '../../data-providers/postgres/models';
-import { LocationTable } from '../../data-providers/postgres/models/location';
+import { createBrandedValue } from '@unocha/hpc-api-core/src/util/types';
+import { InstanceDataOfModel } from '@unocha/hpc-api-core/src/db/util/raw-model';
 
 @Service()
 export class LocationService {
   async findById(
-    models: DbModels,
+    models: Database,
     id: number
-  ): Promise<LocationTable | undefined> {
-    const location = await models.location.getOne(id);
+  ): Promise<InstanceDataOfModel<Database['location']>> {
+    const location = await models.location.findOne({
```

Comment:
`.get()` could be used here instead

Reply:
Wonder where I saw this review comment :smile:

Reply:
😉